Changelog.md

```diff
@@ -282,8 +286,8 @@ Requires Elpi 1.14.1 and Coq 8.15.
 Requires Elpi 1.13.6 and Coq 8.15.
 
 ### APPS
-- `derive Inductive i {A}` now correctly sets `A` implicit status
-- `lock Definition f {A}` now correctly sets `A` implicit status
+- `derive Inductive i {A}` now correctly sets `A` implicit status (XXX)
+- `lock Definition f {A}` now correctly sets `A` implicit status (XXX)
```

Comment:
I can just drop this, but should we add a note here since we found later problems?

Reply:
In doubt, dropped again.